### PR TITLE
Multi-language support: Add zh-CN

### PR DESCRIPTION
**Pull request by zuozishi**
Multi-language support: Add zh-CN

### CODE REVIEW

**Comment by fbigun** [comment]
Very good feature ！:v:

**Comment by zuozishi** [comment]
#345

**Comment by felixse** [comment]
Settings window is crashing, I guess there is some english translation missing. I can have a look at it later.

**Comment by wwb123123** [comment]
Emmm, I didn't test english, sorry.

**Comment by zuozishi** [comment]
Emmm, I didn't test english, sorry.

**Comment by felixse** [comment]
I created a project to track translations here: https://poeditor.com/join/project/TfKNIt826z
Looks like 4 chinese translations are missing

**Comment by zuozishi** [comment]
@felixse 
image

**Comment by felixse** [comment]
Looks good now 👍 
I also added a little script to update translations.

File: UpdateTranslations.ps1 [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Push-Location (Split-Path -Path $MyInvocation.MyCommand.Definition -Parent)
+
+$api_token = "😉";
```
Comment (zuozishi) [inline UpdateTranslations.ps1]: lol

Comment (felixse) [reply UpdateTranslations.ps1]: You can get your API token in POEditor Account Settings. Didn't want to share mine for obvious reasons 😄

**Pull request merged**